dead.py

```diff
@@ -96,9 +105,9 @@ def _filenames(
     out = subprocess.check_output(('git', 'ls-files')).decode()
     for filename in out.splitlines():
         if (
-                not files_re.search(filename) or
-                exclude_re.search(filename) or
-                'python' not in tags_from_path(filename)
+                not files_re.search(filename)
+                or exclude_re.search(filename)
```

Comment:
Pretty sure this was done by one of the pre-commit hooks

Reply:
yeah autopep8 has a bug, please put it back: https://github.com/hhatto/autopep8/issues/456

Reply:
How do I put it back without `pre-commit` changing it back? is there something like `# noqa` that I can add?

Reply:
The bug is that autopep8 will rewrite it only if something else changes, otherwise it is stable